Tests/SwiftCliCoreTests/OffsetUtilityTest.swift

```diff
@@ -0,0 +1,169 @@
+                                       leftOffset: 0)]
+        expect(result).to(equal(expectedResult))
+    }
+
+    func testGatherAllMetaDataTestsLeftInc() throws {
+        let result = OffsetUtility().gatherAllMetaData(stitchArray: [["k1", "k1", "p1"], ["k1", "yo", "k1", "p1"]])
+
+        let expectedResult = [RowInfo(row: ["k1", "k1", "p1"],
+                                      rowNumber: 0,
+                                      bottomLine: "└─┴─┴─┘",
+                                      stitchSymbols: "│ │ │-│\n",
+                                      width: 3,
+                                      leftIncDec: 0,
+                                      rightIncDec: 0,
+                                      leftOffset: 1),
+                              RowInfo(row: ["k1", "yo", "k1", "p1"],
+                                      rowNumber: 1,
+                                      bottomLine: "└─┼─┼─┼─┤\n",
+                                      stitchSymbols: "│ │o│ │-│\n",
+                                      width: 4,
+                                      leftIncDec: 1,
+                                      rightIncDec: 0,
+                                      leftOffset: 0)]
+
+        expect(result).to(equal(expectedResult))
+    }
+
+    func testGatherAllMetaDataTestsMultiLeftInc() throws {
+        let result = OffsetUtility().gatherAllMetaData(stitchArray: [["k1", "k1", "p1"], ["k1", "yo", "k1", "p1"], ["k1", "yo", "k1", "k1", "p1"]])
+
+        let expectedResult = [RowInfo(row: ["k1", "k1", "p1"],
+                                      rowNumber: 0,
+                                      bottomLine: "└─┴─┴─┘",
+                                      stitchSymbols: "│ │ │-│\n",
+                                      width: 3,
+                                      leftIncDec: 0,
+                                      rightIncDec: 0,
+                                      leftOffset: 2),
+                              RowInfo(row: ["k1", "yo", "k1", "p1"],
+                                      rowNumber: 1,
+                                      bottomLine: "└─┼─┼─┼─┤\n",
+                                      stitchSymbols: "│ │o│ │-│\n",
+                                      width: 4,
+                                      leftIncDec: 1,
+                                      rightIncDec: 0,
+                                      leftOffset: 1),
+                              RowInfo(row: ["k1", "yo", "k1", "k1", "p1"],
+                                      rowNumber: 2,
+                                      bottomLine: "└─┼─┼─┼─┼─┤\n",
+                                      stitchSymbols: "│ │o│ │ │-│\n",
+                                      width: 5,
+                                      leftIncDec: 1,
+                                      rightIncDec: 0,
+                                      leftOffset: 0)]
+
+        expect(result).to(equal(expectedResult))
+    }
+
```

Comment:
Shouldn't you technically test the logic for left decreases, right increases, right decreases, both single and multiple here?

Reply:
Added as well! Yes!